package.json

```diff
@@ -72,7 +72,7 @@
         "@js-soft/node-logger": "1.2.0",
         "@js-soft/ts-utils": "^2.3.3",
         "@nmshd/iql": "^1.0.2",
-        "@nmshd/runtime": "5.0.0-alpha.9",
+        "@nmshd/runtime": "5.0.0-alpha.14",
```

Comment:
As I understand it, only the changes from release/v5 are included here, but not the changes from main. This means, for example, that `DeletionRequestSent` or `DeletionRequestRejected` have not yet been added to `ConnectorAttributeDeletionStatus`. We therefore need a second PR on main that incorporates the changes there.

Further things that I have noticed:
- Is there a reason, why the `deletionInfo` occurs in the `GetOwnRepositoryAttributesRequest`, but not in the `GetOwnSharedIdentityAttributesRequest` or `GetPeerSharedIdentityAttributesRequest`, for example? I would expect it to be the other way round. What can our Attribute deletion expert @Milena-Czierlinski say about this?
- Does anyone know why the `parentId` property does not occur in the `ConnectorAttribute`?

Reply:
@Magnus-Kuhn could you take care of these comments?

Reply:
tackeled in #236